src/main/java/picard/sam/SamToFastq.java

```diff
@@ -133,6 +133,9 @@
             "If the original read is shorter than CLIPPING_MIN_LENGTH then the original read length will be maintained.")
     public int CLIPPING_MIN_LENGTH = 0;
 
+    @Option(shortName = "GZIP", doc = "Compress output FASTQ files using gzip and append a .gz extension to the file names.", optional = false)
+    public Boolean COMPRESS_OUTPUTS = false;

```

Comment:
This is confusing for when `OUTPUT_PER_RG` is `false`, since you can specify `*.fastq.gz` and it will automatically write a compressed FASTQ even if `COMPRESS_OUTPUTS=false`.  Perhaps change it to `COMPRESS_PER_RG_OUTPUTS`?  Also, I would omit the short name, or have it include `PER_RG` (`PER_RG_GZIP`)?